### DISCUSSION TITLE
Google Analytics - What is the best way to integrate google analytics in my nextjs app?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#14313] [Help]
Good morning guys!

How is the best way to integrate GA in my app? How are you integrating nextjs and google analytics?

Thanks all!

**Comment by desaijay315** [comment]
https://github.com/vercel/next.js/issues/160#issuecomment-266839584

**Comment by jamesmosier** [comment]
I have follow the [with-google-analytics](https://github.com/vercel/next.js/tree/canary/examples/with-google-analytics) example in the past and has worked well!

**Comment by xyy94813** [comment]
https://github.com/vercel/next.js/discussions/16662
